gapis/api/vulkan/state_rebuilder.go

```diff
@@ -2880,7 +2880,7 @@ func (sb *stateBuilder) createSameBuffer(src BufferObjectʳ, buffer VkBuffer) er
 				pNext,                    // pNext
 				src.Info().CreateFlags(), // flags
 				src.Info().Size(),        // size
-				VkBufferUsageFlags(uint32(src.Info().Usage())|uint32(VkBufferUsageFlagBits_VK_BUFFER_USAGE_TRANSFER_DST_BIT)), // usage
+				VkBufferUsageFlags(uint32(src.Info().Usage())|uint32(VkBufferUsageFlagBits_VK_BUFFER_USAGE_TRANSFER_DST_BIT)|uint32(VkBufferUsageFlagBits_VK_BUFFER_USAGE_TRANSFER_SRC_BIT)), // usage
```

Comment:
We cannot do this in the state_rebuilder. This will change the flags on **all** buffers. We are not allowed to change properties in the initial state block.